contributing.md

```diff
@@ -6,7 +6,13 @@ Start a new [Discussion](https://github.com/tendermint/starport/discussions/new)
 
 Thank you for your contribution!
 
-## Opening pull requests 
+## Feedback / Issues
+
+* Search through Google and [existing open and closed GitHub Issues](https://github.com/tendermint/starport/issues) for the answer first. If you find a relevant topic, please comment on the issue.
```

Comment:
```suggestion
* Before you open an issue, do a web search, and check for [existing open and closed GitHub Issues](https://github.com/tendermint/starport/issues) to see if your question has already been asked and answered. If you find a relevant topic, you can comment on that issue.
```

Reply:
yay for this advice. Instead of using the product name Google, it's handy to just say "do a web search"